ios/RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -117,23 +116,13 @@ - (void)layoutSubviews
 
 - (void)setAnnotations:(NSMutableArray *)annotations
 {
-    _newAnnotations = annotations;
-    [self performSelector:@selector(updateAnnotations) withObject:nil afterDelay:0.5];
+    [self performSelector:@selector(updateAnnotations:) withObject:annotations afterDelay:0.5];
```

Comment:
This method should blow away any existing annotations. Also, is the delay needed anymore? This is the only method that calls `-updatedAnnotations:`. Ideally that method should be inlined here.